grr/server/grr_response_server/gui/ui/components/client/client.ng.html

```diff
@@ -7,8 +7,8 @@ <h1 class="mat-h1">{{client.knowledgeBase.fqdn}}</h1>
       <div class="chips">
         <mat-chip-list>
           <online-chip [lastSeen]="client.lastSeenAt"></online-chip>
-          <mat-chip *ngFor="let label of client.labels">
-            {{label.name}}
+          <mat-chip *ngFor="let label of client.labels" (removed)="removeLabel(label.name)">
```

Comment:
Please add `trackBy` and track elements by label name. Two objects are never `===` unless they are the same instance, so Angular will recreate the DOM upon every change even though labels technically stay the same.